src/hooks/tradingView/useTradingView.ts

```diff
@@ -97,15 +106,30 @@ export const useTradingView = ({
 
       tvWidgetRef.current.onChartReady(() => {
         tvWidgetRef.current?.headerReady().then(() => {
-          if (displayButtonRef && tvWidgetRef.current) {
-            displayButtonRef.current = tvWidgetRef.current.createButton();
-            displayButtonRef.current.innerHTML = `<span>${stringGetter({
-              key: STRING_KEYS.ORDER_LINES,
-            })}</span> <div class="displayOrdersButton-toggle"></div>`;
-            displayButtonRef.current.setAttribute(
-              'title',
-              stringGetter({ key: STRING_KEYS.ORDER_LINES_TOOLTIP })
-            );
+          if (tvWidgetRef.current) {
+            if (orderLineToggleRef) {
+              orderLineToggleRef.current = tvWidgetRef.current.createButton();
+              orderLineToggleRef.current.innerHTML = `<span>${stringGetter({
+                key: STRING_KEYS.ORDER_LINES,
+              })}</span> <div class="displayOrdersButton-toggle"></div>`;
+              orderLineToggleRef.current.setAttribute(
+                'title',
+                stringGetter({ key: STRING_KEYS.ORDER_LINES_TOOLTIP })
+              );
+            }
+            if (isOhlcEnabled && ohlcToggleRef) {
+              const getOhlcTooltipString = tooltipStrings.ohlc;
+              const { title: ohlcTitle, body: ohlcBody } = getOhlcTooltipString({
```

Comment:
this doesn't really follow our tooltip convention (we'd ideally use `WithTooltip`) due to the weird way we add custom buttons to the `TradingView` library